developer-workflow/extension-modules.rst

```diff
@@ -5,13 +5,338 @@
 Standard library extension modules
 ==================================
 
-In this section, we could explain how to write a CPython extension with the C language, but the topic can take a complete book.
+In this section, we could explain how to write a CPython extension with the
+C language, but the topic can take a complete book. We will however explain
+how to add a new extension module to the standard library, e.g., a module
```

Comment:
Please avoid Latin abbreviations: https://devguide.python.org/documentation/style-guide/#use-simple-language

```suggestion
how to add a new extension module to the standard library, for example, a module
```

Reply:
I plead guilty. I didn't read the guidelines...